config/routes.rb

```diff
@@ -1,10 +1,9 @@
 Rails.application.routes.draw do
 
-namespace :api do
-  namespace :v1 do
-    resources :athletes, only: [:index, :show, :update, :create]
-    resources :results, only: [:index]
+  namespace :api do
+    namespace :v1 do
+      resources :athletes, only: [:index, :show, :update, :create, :destroy]
```

Comment:
you are using all the action except edit. ``` except: [:edit] ``` would be a better syntax. 
I know that you are gonna start with edit now so I will let it slide. But I don't wanna see
``` [:index, :show, :update, :create, :destroy, :edit] ``` in the next one! Find another way.

Reply:
Agreed, will fix that on the next PR